### PR TITLE
Fix part #10:  Final UI All topics text

### PR DESCRIPTION
**Pull request by veena14cs**
### Explaination

This PR corresponds to All topics text UI.

### Mock

https://xd.adobe.com/view/bee1fced-33bd-4f69-6a89-2c30fe1c4f65-c8d0/screen/d47aa8de-a7a8-4d25-b011-5baefc7b7098/HP-Home-Page-/

### Screenshot
Screenshot 2019-11-07 at 3 45 54 PM

### CODE REVIEW

File: app/src/main/res/layout/all_topics.xml [hunk 1]
```diff
@@ -16,15 +16,14 @@
...
       android:id="@+id/all_topics_text_view"
-      android:layout_width="match_parent"
+      android:layout_width="wrap_content"
       android:layout_height="wrap_content"
-      android:layout_marginStart="24dp"
-      android:layout_marginTop="16dp"
-      android:layout_marginEnd="24dp"
-      android:layout_marginBottom="16dp"
+      android:layout_marginStart="28dp"
+      android:layout_marginTop="65dp"
```
Comment (nikitamarysolomanpvt) [inline app/src/main/res/layout/all_topics.xml]: Please update to multiples of 4

**Review by nikitamarysolomanpvt — COMMENTED** [review]
PTAL

File: app/src/main/res/layout/all_topics.xml (hunk 1 above)
Comment (rt4914) [reply app/src/main/res/layout/all_topics.xml]: +1

File: app/src/main/res/layout/all_topics.xml [hunk 2]
```diff
@@ -16,15 +16,14 @@
```
Comment (rt4914) [inline app/src/main/res/layout/all_topics.xml:7]: Also,

there is `<data> ... </data>` at top of this file which is not getting used anywhere.
I think it was missed in Low-fi PR, so can your remove that too from here?

File: app/src/main/res/layout/all_topics.xml [hunk 3]
```diff
@@ -16,15 +16,14 @@
...
+      android:layout_width="wrap_content"
       android:layout_height="wrap_content"
-      android:layout_marginStart="24dp"
-      android:layout_marginTop="16dp"
-      android:layout_marginEnd="24dp"
-      android:layout_marginBottom="16dp"
+      android:layout_marginStart="28dp"
+      android:layout_marginTop="65dp"
+      android:layout_marginBottom="24dp"
+      android:fontFamily="sans-serif-medium"
```
Comment (rt4914) [inline app/src/main/res/layout/all_topics.xml:15]: Incorrect font, it should be roboto-medium

**Review by rt4914 — CHANGES_REQUESTED** [review]
PTAL

File: app/src/main/res/layout/all_topics.xml (hunk 3 above)
Comment (veena14cs) [reply app/src/main/res/layout/all_topics.xml:15]: This was suggested by @BenHenning
 See: https://stackoverflow.com/a/34445295/3689782.

File: app/src/main/res/layout/all_topics.xml (hunk 2 above)
Comment (veena14cs) [reply app/src/main/res/layout/all_topics.xml:7]: removed.

File: app/src/main/res/layout/all_topics.xml (hunk 1 above)
Comment (veena14cs) [reply app/src/main/res/layout/all_topics.xml]: done.

File: app/src/main/res/layout/all_topics.xml (hunk 3 above)
Comment (rt4914) [reply app/src/main/res/layout/all_topics.xml:15]: Sounds great, I wa not aware of this similarity.

**Review by rt4914 — APPROVED** [review]
LGTM.

**Review by nikitamarysolomanpvt — APPROVED** [review]
LGTM

**Comment by seanlip** [comment]
Hi @veena14cs -- what would you like me to look at here? If the UI, perhaps showing a screenshot in context would be helpful -- I can't tell much from just looking at the text.

(Also, for UI reviews, it's fine to just loop in @mschanteltc instead rather than me.)

**Comment by veena14cs** [comment]
> Hi @veena14cs -- what would you like me to look at here? If the UI, perhaps showing a screenshot in context would be helpful -- I can't tell much from just looking at the text.
> 
> (Also, for UI reviews, it's fine to just loop in @mschanteltc instead rather than me.)

Ok got it :) thanks.

**Comment by mschanteltc** [comment]
Hi Veena! As mentioned by Sean,  is there something specific that needs checking?

**Pull request merged**